website/package.json

```diff
@@ -50,6 +50,7 @@
     "yarn-audit-fix": "^9.2.1"
   },
   "resolutions": {
+    "htmlnano": "2.0.3",
```

Comment:
Don't think we need it here, the website (docusaurus) doesn't seem to use htmlnano

Reply:
Thanks, good point, will remove it and test the docs.